src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/SimpleRenameStrategy.cs

```diff
@@ -1,36 +1,36 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 using System;
+using System.Globalization;
+using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
-using System.Globalization;
-using System.IO;
-using Microsoft.CodeAnalysis.CSharp;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Rename
 {
     internal sealed class SimpleRenameStrategy : AbstractRenameStrategy
     {
-        private readonly IRoslynServices _roslynServices;
-
-        public SimpleRenameStrategy(IProjectThreadingService threadingService, IUserNotificationServices userNotificationService, IEnvironmentOptions environmentOptions, IRoslynServices roslynServices)
-            : base(threadingService, userNotificationService, environmentOptions)
+        public SimpleRenameStrategy(
+            IProjectThreadingService threadingService,
+            IUserNotificationServices userNotificationService,
+            IEnvironmentOptions environmentOptions,
+            IRoslynServices roslynServices)
+            : base(threadingService, userNotificationService, environmentOptions, roslynServices)
         {
-            _roslynServices = roslynServices;
         }
 
         // For the SimpleRename, it can attempt to handle any situtation
         public override bool CanHandleRename(string oldFileName, string newFileName, bool isCaseSensitive)
         {
             var oldNameBase = Path.GetFileNameWithoutExtension(oldFileName);
             var newNameBase = Path.GetFileNameWithoutExtension(newFileName);
-            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase) && (!string.Equals(Path.GetFileName(oldNameBase), Path.GetFileName(newNameBase), isCaseSensitive?StringComparison.Ordinal:StringComparison.OrdinalIgnoreCase));
+            return _roslynServices.IsValidIdentifier(oldNameBase) && _roslynServices.IsValidIdentifier(newNameBase) && (!string.Equals(Path.GetFileName(oldNameBase), Path.GetFileName(newNameBase), isCaseSensitive?StringComparison.Ordinal:StringComparison.OrdinalIgnoreCase));
```

Comment:
The roslynService delegates to SyntaxFactsService which eventually calls SyntaxFacts - why the indirection?